kevinene91/Store-Manager-API
# #161209936 Sales endpoints

### What does this PR do? 
It has the products endpoints 
` POST /api/v1/sales`
`GET /api/v1/sales`
`GET /api/v1/sales/1`
`PUT /api/v1/sales/1`
`DEL /api/v1/sales/1`
 
### Description of Task completed?
Add Sales endpoint, Attendants should post sales
### How to Manually test this?
 
- Clone the repository `git clone https://github.com/kevinene91/Store-Manager-API.git`
- Change into the app directory `cd Store-Manager-API`
- get this branch code `git fetch origin ft-store-sales-161209936`
- Navigate the folder structure
- create a virtual environment `python3 -m pyvenv venv`
- pip install -r requirements.txt
- export SECRET_KEY ="random string here"
-  run `python run.py`
- Test on Postman 
- Navigate folder structure 
app/tests/v1 run `pytest test_sales.py` should pass 
### Relevant PVT story? 

[161209936](https://www.pivotaltracker.com/n/projects/2203367/stories/161209936)
